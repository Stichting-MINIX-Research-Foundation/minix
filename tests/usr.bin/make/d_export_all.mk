# $NetBSD: d_export_all.mk,v 1.1 2012/03/17 16:33:14 jruoho Exp $

UT_OK=good
UT_F=fine

.export

.include "d_export.mk"

UT_TEST=export-all
UT_ALL=even this gets exported
