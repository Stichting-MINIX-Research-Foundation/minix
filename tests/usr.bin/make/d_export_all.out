UT_ALL=even this gets exported
UT_DOLLAR=This is $UT_FU
UT_F=fine
UT_FOO=foobar is fubar
UT_FU=fubar
UT_NO=all
UT_OK=good
UT_TEST=export-all
UT_ZOO=hoopie
